microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+
+#### Experimental features 
+
+*  Added support for the Experimental File API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+* [CoreWebView2ExperimentalFile interface](#)
+   * [CoreWebView2ExperimentalFile::get_Path method](#)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+* [CoreWebView2ExperimentalFile interface](#)
+   * [CoreWebView2ExperimentalFile::get_Path method](#)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalFile interface](#)
+   * [ICoreWebView2ExperimentalFile::get_Path method](#)
+* [ICoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+---
+
+*  Added support for Experimental Object Collection View API
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [ICoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [ICoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+---
+
+#### Promotions
+
+The following APIs are promoted to stable in this prerelease SDK:
+
+*  The SharedBuffer API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2_17 interface](#)
+	* [CoreWebView2_17::PostSharedBufferToScript method](#)
+* [CoreWebView2Environment12 interface](#)
+	* [ICoreWebView2Environment12:: CreateSharedBuffer method](#)
+* [CoreWebView2Frame4 interface](#)
+	* [CoreWebView2Frame4:: PostSharedBufferToScript method](#)
+* [CoreWebView2SharedBuffer interface](#)
+	* [CoreWebView2SharedBuffer:: OpenStream method](#)
+	* [CoreWebView2SharedBuffer:: Close method](#)
+	* [CoreWebView2SharedBuffer:: get_Size method](#)
+	* [CoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [CoreWebView2SharedBuffer:: get_FileMappingHandle method](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2_17 interface](#)
+	* [CoreWebView2_17::PostSharedBufferToScript method](#)
+* [CoreWebView2Environment12 interface](#)
+	* [CoreWebView2Environment12:: CreateSharedBuffer method](#)
+* [CoreWebView2Frame4 interface](#)
+	* [CoreWebView2Frame4:: PostSharedBufferToScript method](#)
+* [CoreWebView2SharedBuffer interface](#)
+	* [CoreWebView2SharedBuffer:: OpenStream method](#)
+	* [CoreWebView2SharedBuffer:: Close method](#)
+	* [CoreWebView2SharedBuffer:: get_Size method](#)
+	* [CoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [CoreWebView2SharedBuffer:: get_FileMappingHandle method](#)

```

Comment:
```suggestion
* [CoreWebView2 Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
   * [CoreWebView2.PostSharedBufferToScript Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#postsharedbuffertoscript)
* [CoreWebView2Environment Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
   * [ICoreWebView2Environment.CreateSharedBuffer Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#createsharedbuffer)
* [CoreWebView2Frame Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
   * [CoreWebView2Frame.PostSharedBufferToScript Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#postsharedbuffertoscript)
* [CoreWebView2SharedBuffer Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2sharedbuffer?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
   * [CoreWebView2SharedBuffer.OpenStream Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2sharedbuffer?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#openstream)
   * [CoreWebView2SharedBuffer.Close Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2sharedbuffer?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#close)
   * [CoreWebView2SharedBuffer.Size Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2sharedbuffer?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#size)
   * [CoreWebView2SharedBuffer.Buffer Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2sharedbuffer?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#buffer)
   * [CoreWebView2SharedBuffer.FileMappingHandle Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2sharedbuffer?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#buffer)<!--todo: n/a, omit link?-->
```
tech review: omit the final item for WinRT: FileMappingHandle?

Reply:
instance 3
@peiche-jessica - find "todo", include or omit this link?

Reply:
Omit from WinRT.

Reply:
Resolved; omitted.